### PR TITLE
Enhance rebuild_boot_iso script

### PR DESCRIPTION
**Pull request by jkonecny12**
Tweak the script:
- Add help
- Add support for additional RPM files
- Make script failures fatal
- Move them to scripts/testing
- Small fix in "warming sudo" print

### CODE REVIEW

**Comment by jkonecny12** [comment]
A question. Do we enable Live ISO for this script or add a new one for the Live ISO?

**Comment by adamkankovsky** [comment]
Maybe it would be a good idea to add an option for webui as well.

File: scripts/testing/rebuild_boot_iso [hunk 1]
```diff
@@ -50,6 +54,15 @@ rm -f ${BOOT_ISO_GIT_REVISION}
 mkdir -p result/iso/
 # note the Git revision from which we build the boot.iso
 git rev-parse HEAD > result/iso/boot.iso.git_rev
+
+# build the anaconda rpms
 make -f ./Makefile.am container-rpms-scratch
+
+# copy additional web UI packages
```
Comment (M4rtinK) [inline scripts/testing/rebuild_boot_iso:67]: Couldn't this be in principle any RPMs ?

**Review by M4rtinK — APPROVED** [review]
Looks good to me - just a small wording suggestion. :)

File: scripts/testing/rebuild_boot_iso (hunk 1 above)
Comment (jkonecny12) [reply scripts/testing/rebuild_boot_iso:67]: yes it will take any RPM

**Comment by jkonecny12** [comment]
> Maybe it would be a good idea to add an option for webui as well.

You mean boot.iso with Web UI?

File: scripts/testing/rebuild_boot_iso [hunk 2]
```diff
@@ -21,26 +21,31 @@ Usage:
 
 Options:
  -h, --help                         Print this help
+ -a, --add-rpms                     Add additional RPMs (for example anaconda-webui package)
+                                    these packages needs to be already requested in ISO
```
Comment (M4rtinK) [inline scripts/testing/rebuild_boot_iso:26]: I think there is a typo - should be just `"need"`. Also I suggest `"already requested to be in the ISO"`

File: scripts/testing/rebuild_boot_iso (hunk 1 above)
Comment (M4rtinK) [reply scripts/testing/rebuild_boot_iso:68]: Then I suggest changing the single line comment - `"# copy additional RPM packages"`

**Review by M4rtinK — APPROVED** [review]

**Pull request merged**